datasets/superb/README.md

```diff
@@ -279,8 +325,8 @@ An example from each split looks like:
 
 #### si
 
-[More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
-
+- `file` (`string`): Path to the WAV audio file.
+- `label` (`ClassLabel`): Label (ID) of the speaker.
```

Comment:
Would it be relevant to explain the mapping between the label id (int) and label name (str like "idxxxx") here ? or maybe not ?